search-app/resources/templates/eosdis-landing-links.html

```diff
@@ -0,0 +1,9 @@
+{% extends "templates/base.html" %}
+
+{% block title %}CMR Search - Landing Pages for EOSDIS Collections{% endblock %}
```

Comment:
Oh, you know what, these aren't actually landing pages and shouldn't be called landing pages -- they are pages for _links_ to landing pages ... I'll double check the terminology in the templates. Might be worth some clarifying text on the pages themselves.